src/utils/perfherder/chartJs/perfherderFormatter.js

```diff
@@ -35,11 +37,30 @@ const generateInitialOptions = series => {
             ).toFixed(2);
 
             tooltipData.push(`Δ ${delta} (${deltaPercentage}%)`);
+
+            const currRevision = series.data[tooltipItems[0].index].revision;
+            const prevRevision =
+              series.data[tooltipItems[0].index - 1].revision;
+            const hgLink = `https://hg.mozilla.org/mozilla-central/pushloghtml?fromchange=${prevRevision}&tochange=${currRevision}`;
+
+            tooltipData.push(
+              `<a href="${hgLink}" target="_blank">${currRevision.slice(
+                0,
+                12
+              )}</a>`
+            );
           }
 
           return tooltipData;
         },
       },
+      enabled: false,
+      custom(tooltipModel) {
+        // eslint-disable-next-line no-underscore-dangle
+        const { canvas } = this._chart;
+
+        generateCustomTooltip(canvas, tooltipModel);
+      },
```

Comment:
these lines should read 

    custom: CustomTooltip

Reply:
Yes, leave it until later.  I see that the `tooltipModel` does not have all the information we require, we must attach some more before we call `CustomTooltip`

Reply:
No it hasn't, but there's a workaround, check my other comment please.